src/main/java/Main.java

```diff
@@ -0,0 +1,15 @@
+import fruitstore.FruitShopServeImpl;
+import fruitstore.ShopServe;
+import java.io.IOException;
+
+public class Main {
```

Comment:
The project should have packages (service, db, model, strategy...)

package service should have: FruitShopService, ParserService, ReaderService, ReportService, WriterService
package db should have: Storage
package model should have: FruitTransaction
package strategy should have: OperationHandler, OperationStrategy

all implementation should be to move to package impl

this will be a good code decomposition. 